spec/lib/conjur_client_spec.rb

```diff
@@ -125,4 +125,35 @@
       end
     end
   end
+
+  describe "version" do
+    before do
+      allow(ENV).to receive(:[]).and_call_original
+    end
+
+    it "uses the default version (5) when the input is an empty string" do
+      allow(ENV).to receive(:[]).with('CONJUR_VERSION').and_return("")
+      expect(ConjurClient.version).to eq(5)
+    end
+
+    it "uses the default version (5) when the input is empty" do
+      allow(ENV).to receive(:[]).with('CONJUR_VERSION')
+      expect(ConjurClient.version).to eq(5)
+    end
+
+    it "uses the default version (5) when the input is invalid" do
```

Comment:
@BradleyBoutcher just seeing this and this case is actually "it raises a runtime error when the input is invalid"